olp-cpp-sdk-core/tests/thread/TaskContinuationTest.cpp

```diff
@@ -24,54 +24,50 @@
 #include <gtest/gtest.h>
 
 #include <olp/core/client/HttpResponse.h>
-#include <olp/core/client/OlpClientSettingsFactory.h>
 #include <olp/core/thread/TaskContinuation.h>
+#include <olp/core/thread/ThreadPoolTaskScheduler.h>
 
 const std::string BAD_REQUEST_MESSAGE = "Bad Request";
 
+template <typename Response>
+using ResponseType = olp::client::ApiResponse<Response, olp::client::ApiError>;
+
 class TaskContinuationFixture : public ::testing::Test {
  public:
   void SetUp() override {
-    scheduler =
-        olp::client::OlpClientSettingsFactory::CreateDefaultTaskScheduler();
+    scheduler = std::make_unique<olp::thread::ThreadPoolTaskScheduler>(1);
```

Comment:
```suggestion
    scheduler = std::make_shared<olp::thread::ThreadPoolTaskScheduler>(1);
```

Also please use the default scheduler instead

Reply:
done